kdebase-3.4.92.orig/kdm/kfrontend/kdm_greet ve kdebase-3.4.92/kdm/kfrontend/kdm_greet dosyalar覺 birbirinden farkl覺
diff -ur kdebase-3.4.92.orig/kdm/kfrontend/kgreeter.cpp kdebase-3.4.92/kdm/kfrontend/kgreeter.cpp
--- kdebase-3.4.92.orig/kdm/kfrontend/kgreeter.cpp	2005-11-10 00:46:01.000000000 +0200
+++ kdebase-3.4.92/kdm/kfrontend/kgreeter.cpp	2005-11-10 01:42:58.000000000 +0200
@@ -832,6 +832,7 @@
 #endif
 
 	system_button = themer->findNode( "system_button" );
+	shutdown_button = themer->findNode( "shutdown_button" );
 	QAccel *accel = new QAccel( this );
 	accel->insertItem( ALT+Key_M, 0 );
 	connect( accel, SIGNAL(activated( int )), SLOT(slotActionMenu()) );
@@ -922,6 +923,8 @@
 		slotSessMenu();
 	else if (id == "system_button")
 		slotActionMenu();
+	else if (id == "shutdown_button")
+		slotShutdown(-1);
 }
 
 void
diff -ur kdebase-3.4.92.orig/kdm/kfrontend/kgreeter.h kdebase-3.4.92/kdm/kfrontend/kgreeter.h
--- kdebase-3.4.92.orig/kdm/kfrontend/kgreeter.h	2005-11-10 00:46:01.000000000 +0200
+++ kdebase-3.4.92/kdm/kfrontend/kgreeter.h	2005-11-10 01:43:07.000000000 +0200
@@ -153,7 +153,7 @@
 	KdmThemer *themer;
 	KdmItem *caps_warning, *xauth_warning, *pam_error, *timed_label,
 	        *console_rect, *userlist_rect,
-	        *session_button, *system_button;
+	        *session_button, *system_button, *shutdown_button;
 
   public: // from KGVerifyHandler
 	virtual void verifyFailed();
kdebase-3.4.92.orig/kdm/kfrontend/kgreeter.o ve kdebase-3.4.92/kdm/kfrontend/kgreeter.o dosyalar覺 birbirinden farkl覺
